src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -1076,7 +1141,8 @@ def _get_keyvault_client(cli_ctx):
     from azure.keyvault import KeyVaultClient, KeyVaultAuthentication
 
     def _get_token(server, resource, scope):  # pylint: disable=unused-argument
-        return Profile(cli_ctx=cli_ctx).get_login_credentials(resource)[0]._token_retriever()  # pylint: disable=protected-access
+        return Profile(cli_ctx=cli_ctx).get_login_credentials(resource)[
+            0]._token_retriever()  # pylint: disable=protected-access
```

Comment:
Again, please back out the format change here and below. The change here is not correct anyway.

Reply:
Backed out